package.json

```diff
@@ -48,6 +48,7 @@
         "notebookEditorEdit"
```

Comment:
For publish to work, don't forget to update the package.json version number. Or change build-test.yml to generate a new one every build.